src/views/flow_viewer_details_vw.sql

```diff
@@ -0,0 +1,139 @@
+/* 
+-- Flows for APEX - flow_viewer_details_vw.sql
+-- 
+-- (c) Copyright Flowquest Limited, 2022.
+--
+-- Created  23-Jun-2024 Richard Allen Flowquest Limited. 
+--
+*/
+--create or replace view flow_viewer_details_vw
+--as
+
+
+
+with completed_objects as (
+        select distinct sflg.sflg_prcs_id             as prcs_id
+                      , sflg.sflg_dgrm_id             as dgrm_id
+                      , sflg.sflg_diagram_level       as diagram_level
+                      , sflg.sflg_objt_id             as objt_id
+                      , sflg.sflg_sbfl_iteration_path as iter_path
+                      , sflg.sflg_step_key            as step_key
+          from flow_subflow_log sflg
+         where sflg.sflg_objt_id not in (   
+                                          select sbfl.sbfl_current
+                                            from flow_subflows sbfl
+                                           where sbfl.sbfl_prcs_id = sflg.sflg_prcs_id
+                                             and sbfl.sbfl_diagram_level = sflg.sflg_diagram_level
+                                             and sbfl.sbfl_iteration_path = sflg.sflg_sbfl_iteration_path
+                                             and sbfl.sbfl_current is not null
+                                        )
+), all_completed as ( 
+    select prcs_id, dgrm_id, diagram_level, iter_path,
+           listagg( objt_id, ':') within group (order by objt_id) as bpmn_ids
+    from completed_objects
+    group by prcs_id, dgrm_id, diagram_level, iter_path
+), all_current as (
+  select sbfl_prcs_id as prcs_id
+       , sbfl_dgrm_id as dgrm_id
+       , sbfl_diagram_level as diagram_level 
+       , sbfl_iteration_path as iter_path
+       , listagg(sbfl_current, ':') within group ( order by sbfl_current ) as bpmn_ids
+    from flow_subflows
+   where sbfl_current is not null  
+group by sbfl_prcs_id, sbfl_dgrm_id, sbfl_diagram_level, sbfl_iteration_path
+), all_errors as (
+  select sbfl_prcs_id as prcs_id
+       , sbfl_dgrm_id as dgrm_id
+       , sbfl_diagram_level as diagram_level
+       , sbfl_iteration_path as iter_path
+       , listagg(sbfl_current, ':') within group (order by sbfl_current) as bpmn_ids
+    from flow_subflows
+   where sbfl_current is not null
+     and sbfl_status = 'error'
+group by sbfl_prcs_id, sbfl_dgrm_id, sbfl_diagram_level, sbfl_iteration_path
+), iteration_paths as (
```

Comment:
should we slim this down for community edition?

Reply:
file / view should be deleted